Hestia/Hestia/AppDelegate.cs

```diff
@@ -50,7 +55,7 @@ public bool IsServerValid()
         public bool IsAuth0LoginValid()
         {
             //TODO possibly a backend method that checks if token is still valid
-            return true;
+            return false;
         }
```

Comment:
I'm not sure how useful this is, seeing as how the user would just have to login again then.

Reply:
It is for the case that a token is saved as a UserDefaults, but then the token is removed/made invalid by the Webserver. Then it cannot use the token anymore.

Reply:
I think for now we can just assume this will never happen. Seeing as how the web server also does not have an implementation for this.